openapi-cli/src/test/java/org/ballerinalang/ballerina/openapi/convertor/service/OpenApiConverterUtilsTest.java

```diff
@@ -200,7 +200,7 @@
             "   }\n" +
             "}";
 
-    @Test(enabled = false, description = "Test ballerina to openApi conversion")
+    @Test(enabled = true, description = "Test ballerina to openApi conversion")
```

Comment:
Just removing entire enabled attribute would make it enable right? Shall we do that for all the test methods?